client/options.go

```diff
@@ -298,6 +305,26 @@ func (o *options) Validate() error {
 	if o.readerIteratorAllocate == nil {
 		return errNoReaderIteratorAllocateSet
 	}
+	if err := topology.ValidateConsistencyLevel(
```

Comment:
Might be nice to have the topology expose a "ValidateAllConsistencyLevels" method?

Reply:
Hm, I think it's a little cleaner like this because we may want to return the specialization of which consistency level is invalid.